lib/config/migrations/custom/require-config-migration.ts

```diff
@@ -0,0 +1,14 @@
+import { AbstractMigration } from '../base/abstract-migration';
+
+export class RequireConfigMigration extends AbstractMigration {
+  override readonly propertyName = 'requireConfig';
+
+  override run(value: unknown): void {
+    if (value === true) {
+      this.rewrite('required');
+    }
+    if (value === false) {
+      this.rewrite('optional');
+    }
```

Comment:
```suggestion
    if (value === false) {
      this.rewrite('optional');
    } else {
      this.rewrite('required');
    }
```

i think we should always migrate it, we only get to that function if the property `requireConfig` is defined, but value can be invalid